tests/integrations/test_pipeline_registration.py

```diff
@@ -1,13 +1,27 @@
 import pytest
 from typer.testing import CliRunner
 
+from garden_ai import GardenClient
 from garden_ai.app.main import app
 from garden_ai.utils.filesystem import load_pipeline_from_python_file
 from tests.fixtures.helpers import get_fixture_file_path  # type: ignore
 
 runner = CliRunner()
 
 
+@pytest.fixture(autouse=False)
+def do_not_set_mlflow_env_variables():
+    # actually, DO set mlflow env variables
+    # locally defining a fixture to override
+    # the one mocking `setup_mlflow_env`
+    pass
```

Comment:
This feels kludgey (and was initiated by me, you're just seeing it to its logical conclusion). It's too bad that to set MLFlow variables, the function that needs the override has to use something called `do_not_set_mlflow_env_variables`. Some very quick Googling didn't turn up canonical ways to do setup/teardown for pytest outside of the fixture system so I don't really have an alternative to suggest atm. But I'd like us both to keep our eyes on this and get to something more intuitve.